### ISSUE TITLE
I added a club, then couldn't find it on the map

### ISSUE THREAD

**Issue by adamlofting** [mozilla/teach.webmaker.org#364]
1. I added a test club, just using a city name ('Guildford') 
1. It seemed to work, 
1. But on clicking 'take me to my club' I couldn't see it. Or find it on the map afterwards.

For a user, this would be pretty confusing.

In the teach API, it ended up as:

```JSON
 {
        "url": "https://teach-api.herokuapp.com/api/clubs/57/",
        "name": "Creative Club",
        "website": "http://what.com",
        "description": "Teaching this that and the other.",
        "location": "Guildford",
        "latitude": null,
        "longitude": null,
        "owner": "adam-lofting"
    }
```

The `null`s seem to be the issue. Can we validate for this?

**Comment by adamlofting** [comment]
cc/ @toolness @alicoding to see who this is best assigned to?

**Comment by toolness** [comment]
Thanks! This is going to be resolved ASAP with #353! I'm marking it as a duplicate, hope that is ok.

**Comment by adamlofting** [comment]
:+1:
